packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -490,17 +506,10 @@ class Auth {
                     !!refreshTokenGuest
                 )
             } catch (error) {
-                // If the refresh token is invalid, we need to re-login the user
-                if (error instanceof Error && 'response' in error) {
-                    // commerce-sdk-isomorphic throws a `ResponseError`, but doesn't export the class.
-                    // We can't use `instanceof`, so instead we just check for the `response` property
-                    // and assume it is a fetch Response.
-                    const json = await (error['response'] as Response).json()
-                    if (json.message === 'invalid refresh_token') {
-                        // clean up storage and restart the login flow
-                        this.clearStorage()
-                    }
-                }
+                // If the refresh token login fails, we fall back to the new guest refresh login
+                await this.logTokenRequestError(error as Error)
+                // clean up storage and restart the login flow
+                this.clearStorage()
```

Comment:
Minor: how about us console.loging here to tell others that we've handled the error by doing a guest login? I think it'd be useful for them to see that in the log too... not just the errors.